src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/AspNetCoreInstrumentationEventSource.cs

```diff
@@ -28,45 +28,45 @@ internal sealed class AspNetCoreInstrumentationEventSource : EventSource
         public static AspNetCoreInstrumentationEventSource Log = new();
 
         [NonEvent]
-        public void RequestFilterException(Exception ex)
+        public void RequestFilterException(string handlerName, string eventName, string operationName, Exception ex)
```

Comment:
I think the OperationName is always the same as we always process a very particular activity in the instrumentation library.